driver-core/src/test/java/com/datastax/driver/core/UnsetValuesTest.java

```diff
@@ -0,0 +1,165 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.core;
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.testng.Assert.fail;
+
+import static com.datastax.driver.core.ProtocolVersion.V3;
+import static com.datastax.driver.core.ProtocolVersion.V4;
+
+public class UnsetValuesTest {
```

Comment:
There are two obsolete test methods `PreparedStatement#unbound*` (I'm fine replacing them with this dedicated test).

For the sake of performance we should avoid dedicated ccm instances and use `CCMBridge.PerClassSingleNodeCluster` as much as possible. In this test we could achieve this by building a separate `Cluster` instance when we want to force the protocol version down.